src/minimega/qos_cli.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"fmt"
+	"minicli"
+	"strconv"
+)
+
+var qosCLIHandlers = []minicli.Handler{
+	{
+		HelpShort: "add qos constraints to an interface",
+		HelpLong:  "Fritz Fritz Fritz",
+		Patterns: []string{
+			"qos list",
+			"qos <add,> <interface> <loss,> <percent>",
+			"qos <add,> <interface> <delay,> <ms>",
+			"qos <add,> <interface> <loss,> <percent> <delay,> <ms>",
```

Comment:
It looks like the original reason for supporting stacking here is that tc requires all of the config to happen in a single call to tc. It looks like the implementation could support ditching the stacked solution here and instead allowing the user to issue 2 commands (loss, then delay) and have tc be called with any existing config on a given tap:

    qos add mega_tap0 loss 50%
    qos add mega_tap0 delay 50ms

now mega_tap0 should have both because the second command still knew in the qos.params map about the loss setting.

So I would get rid of the stacked API.